api/src/services/authentication.ts

```diff
@@ -128,6 +128,9 @@ export class AuthenticationService {
 			user_agent: userAgent,
 		});
 
+		const now = Date.now();
```

Comment:
One more question:

`Date.now()` generates a number, eg `1615566339809`. Does that work cross-database? I'm wondering if a database might throw because type number doesn't match a date

Reply:
Doesn't knew handle it 🤔

Reply:
Ehhhhhhhh you'd think so, but I don't want to bet on it 😬 Lets double check that

Reply:
Great catch.....Changed it to new Date().. I checked it in PG and MariaDB it works now...